docs/keto/guides/userset-rewrites.mdx

```diff
@@ -1,56 +1,59 @@
 ---
 id: userset-rewrites
-title: Define global permission rules using Ory Permission Language
-sidebar_label: Global rules
+title: Define global permission rules using the Ory Permission Language
+sidebar_label: Using the Ory Permission Language
 ---
 
 import CodeFromRemote from "@theme/CodeFromRemote"
+import BrowserWindow from "@site/src/theme/BrowserWindow"
 const file = (path) => `https://github.com/ory/keto/blob/master/contrib/rewrites-example/${path}`
 
-This guide explains how to configure namespaces and relations using the
-[Ory Permission Language](../reference/ory-permission-language).
+This guide explains how to configure namespaces and relationship rules using the
+[Ory Permission Language](../reference/ory-permission-language) (OPL). You can then run fine-grained checks against Ory Keto,
+which are answered based on a combination of OPL and the concrete high-level relationships stored.
```

Comment:
> You can then run fine-grained checks against Ory Keto
If you start from scratch, you won't be doing anything against Keto until you complete the optional step which is running a Keto instance, correct? The permission checks are handled by Ory Permissions in Ory Network if I'm not mistaken.

Reply:
Well you don't have to do anything if you use the Ory Network. If you want to self-host, there's a section below.